website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,267 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another Windows Subsystem for Linux (WSL) distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, [Podman Desktop configures Podman in a dedicated Windows Subsystem for Linux (WSL) distribution: the Podman Machine](/docs/podman/creating-a-podman-machine.md).
+It also configures the Windows Podman client to communicate with the Podman Machine.
+However, it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the most common context to walk you through the steps to configure your WSL distribution:
+
+- Ubuntu distribution of Linux.
+- Default Podman Machine.
+
+In foldable details, you can find alternative steps for least common contexts:
+
+- Custom WSL distribution.
+- Custom Podman Machine.
+
+## Configuring your WSL distribution
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. To communicate with the remote Podman Machine, you need a Podman client.
+
+   Install the Podman client:
+
+   ```shell-session
+   $ sudo apt update
+   $ sudo apt install podman
```

Comment:
I did nt know about this `podman-remote` alternative. Honestly I find it a bit confusing to have to change of binary. People already have a tendency to do `alias docker=podman`... From a UX point of view, I think I love if the connection had a remote attribute in order to specify it when setting up the connection once for all.

Probably something like
```podman system connection add --remote <name> <socket-path>```

Reply:
Reading these comments, it seems indeed that the features in `podman-remote` make the case strong enough to cover `podman-remote` rather than the (old) `podman` package.

We might keep mention of the installation with the package for environments where	installing binaries must be avoided.

Reply:
To check:

Is it posssible to run:

```
podman-remote system connection add
```

Or do we still need `podman` to define the connection?

Reply:
Yes you can define the connection with `podman-remote` the same way as with the full version of `podman`. That's how it's configured in the doc I linked to above.

Reply:
Recent features + consistent across distros + one step less. It might be enough to convince people who would prefer using the package.